### DISCUSSION TITLE
Chromescat/airplay support in videojs

### DISCUSSION THREAD

**Discussion by mustafaaden98** [videojs/video.js#7800] [General]
Hi, 
How can I add chromecast/airplay button in control bar of videojs ?
Thanks for your help!

**Comment by gkatsev** [comment]
Take a look at these two plugins:
https://github.com/silvermine/videojs-airplay
https://github.com/silvermine/videojs-chromecast

**Comment by mustafaaden98** [comment]
Hi Gary, I have checked that, but those two have some serious
vulnerabilities and also those are not actively maintained.
Do you know some other good alternatives?

Thanks
Mustafa Adenwala

**Reply by gkatsev** [reply]
I see changes in the repo recently. Not really aware of alternatives.
What vulnerabilities are you seeing? If it's warnings from `npm audit`, most if not all of these don't apply to the project since it's client-side.
